src/components/Bookings/Bookings.jsx

```diff
@@ -1,11 +1,25 @@
 import Search from "@/components/Search/Search";
 import SearchResult from "../SearchResult/SearchResult";
-import FakeBookings from "@/data/fakeBookings.json";
-import { useState } from "react";
+import { useState, useEffect } from "react";
 
 const Bookings = () => {
-  const [bookings, setBookings] = useState(FakeBookings);
+  const [bookings, setBookings] = useState([]);
 
+  useEffect(() => {
+    fetch("https://nw6-cyf-hotel.glitch.me/fakebookings")
+      .then((res) => {
+        if (!res.ok) {
+          throw new Error(`${res.status}: ${getReasonPhrase(res.status)}`);
+        }
+        return res.json();
+      })
+      .then((data) => setBookings(data))
+      .catch((error) => {
+        setFetchError(error);
+        console.log(error);
+      });
+  }),
```

Comment:
Looking good! I love the attention to error handling. It's so important in the real world!

Reply:
Thank you